checks/collector.py

```diff
@@ -137,13 +143,15 @@ def run(self, checksd=None, start_event=True):
         self.run_count += 1
         log.debug("Starting collection run #%s" % self.run_count)
 
+        if checksd:
+            self.initialized_checks_d = checksd['initialized_checks'] # is of type {check_name: check}
```

Comment:
While you're at it can you update the comments ? 
```python
checksd['initialized_checks']
```
is a list of AgentCheck instances